pkg/virt-handler/virtwrap/manager.go

```diff
@@ -328,21 +327,14 @@ func VMNamespaceKeyFunc(vm *v1.VM) string {
 	return domName
 }
 
-func (l *LibvirtDomainManager) SyncVM(vm *v1.VM) (*api.DomainSpec, error) {
-	var wantedSpec api.DomainSpec
-	mappingErrs := model.Copy(&wantedSpec, vm.Spec.Domain)
-
-	if len(mappingErrs) > 0 {
-		return nil, errors.NewAggregate(mappingErrs)
-	}
-
+func (l *LibvirtDomainManager) SyncVM(vm *v1.VM, domDesign *designer.DomainDesign) (*api.DomainSpec, error) {
```

Comment:
Sure it can. Like Pods have [Volumes](https://kubernetes.io/docs/concepts/storage/volumes/), where one of the volume types is PVC, we will have the same for disks on the VM. PVCs cover a subset of of Volumes on the pod (the ones which work over network, excluding node centric ones, like emptydir, flexvolume or hostdir). It is perfectly fine to directly reverence an iscsi connection from a VM without a PVC in-between, and therefore a mapping from PVC to "Disk" on the VM will always work.